### DISCUSSION TITLE
滚轮某些情况下不起作用

### DISCUSSION THREAD

**Discussion by dd-219** [Caldis/Mos#653] [Q&A]
大佬好，我最近电脑更新了 marginnote4 这个软件，在使用软件的过程中时常出现滚轮无法滚动文档的情况，一开始我以为是 marginnote 软件的原因，后面有在其他软件（ishot）中提到，使用 ishot 进行长截图时，如果 mos 也开了，可能无法用软件进行长截图（https://www.better365.cn/ishot/scrollscreenshot.html），不知道这这些软件有没有共性冲突的地方，烦请大佬有时间解决一下～

**Comment by ethanbbbbbb** [comment]
我也是遇到了这个问题，请问你解决了么
